src/store/messages/saga.ts

```diff
@@ -599,3 +601,18 @@ function* readReceiptReceived({ payload }) {
     }
   }
 }
+
+export function* loadAttachmentDetails(action) {
+  console.log('XXXaction', action);
+  const { media, messageId } = action.payload;
+  // if in progress then don't fetch
```

Comment:
Add something here like:
`yield put(receiveMessage({ id: messageId, media: { ...media, isFetching: true } }));`

or possibly use existing message fetching status, will need to check